armi/reactor/assemblyParameters.py

```diff
@@ -230,7 +230,7 @@ def _enforceNotesRestrictions(self, value):
             "crWithdrawnElevation",
             units="cm",
             description=(
-                "The initial starting elevation of the moveable section of a control rod assembly when fully withdrawn.  Note that this should "
+                "The elevation of the bottom of the bottom-most moveable section of a control rod assembly when fully withdrawn. Note that this should "
```

Comment:
The reason this said starting is because the direction is based on interpretation of the parameter within the context of it's definition (think rods that insert from the bottom of the core). In most cases this is the interpretation though but others may implement onto this more generically. I'm okay with this but wanted to provide some background

Reply:
Oh, yes, you're totally correct. I will change!

Reply:
Okay, what about now?https://github.com/terrapower/armi/pull/1161/commits/0e42c0801059a84ff2270b2a38038bc9c71329bb

Reply:
Nice! I'll merge